buddychatcli/Person.cs

```diff
@@ -0,0 +1,30 @@
+using System;
+using System.Collections.Generic;
+
+using CommandLine;
+
+namespace BuddyChatCLI
+{
+
+    public class Person
+    {
+        
+        public string username { get; set; }
+
+        public string name { get; set; }
+
+        public string pronoun { get; set; }
+
+        public bool audioChatOnly { get; set; }
+
+        public string intro { get; set; }

```

Comment:
I was thinking intro would also be part of dictionary of metadata with key="intro" and value=<intro text>